test/built-ins/Map/prototype/getOrInsertComputed/does-not-have-mapdata-internal-slot-set.js

```diff
@@ -13,7 +13,6 @@ info: |
   2. Perform ? RequireInternalSlot(M, [[MapData]])
   ...
 features: [Set, arrow-function, upsert]
-flags: [noStrict]
 ---*/
 assert.throws(TypeError, function () {
   Map.prototype.getOrInsertComputed.call(new Set(), 1, () => 1);
```

Comment:
The set, and in the test below also the map, should be created outside the `assert.throws` callback